authorization/src/main/kotlin/com/ritense/authorization/web/PermissionResource.kt

```diff
@@ -32,6 +33,7 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
+//@SkipComponentScan
```

Comment:
Why is this one commented out? Does this rely on component scanning?

Reply:
That was unintended. Fixed.